src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/Commands/GetAzureApiManagementGroup.cs

```diff
@@ -15,11 +15,12 @@
 namespace Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Commands
 {
     using System;
+    using System.Collections.Generic;
     using System.Management.Automation;
     using Microsoft.Azure.Commands.ApiManagement.ServiceManagement.Models;
 
-    [Cmdlet(VerbsCommon.Get, "AzureRmApiManagementGroup", DefaultParameterSetName = GetAll)]
-    [OutputType(typeof(PsApiManagementGroup))]
+    [Cmdlet(VerbsCommon.Get, Constants.ApiManagementGroup, DefaultParameterSetName = GetAll)]
+    [OutputType(typeof(IList<PsApiManagementGroup>))]
```

Comment:
If this cmdlets can return both a single object and a list of objects, you should include both in the OutputType attributes.